modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/MarkedItems/CmsKitMarkedItemOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using JetBrains.Annotations;
+using System.Collections.Generic;
+
+namespace Volo.CmsKit.MarkedItems;
+
+public class CmsKitMarkedItemOptions
+{
+    [NotNull]
+    public List<MarkedItemEntityTypeDefinition> EntityTypes { get; } = new();
```

Comment:
What if I define multiple definitions for the same EntityType? Since it's queried by `Single()` it throws an exception but a more clear exception can be thrown